.jenkins/actions/run_standalone.sh

```diff
@@ -45,17 +46,25 @@ TIMESTEPS=60
 RANKS=6
 BENCHMARK_DIR=${ROOT_DIR}/examples/standalone/benchmarks
 DATA_DIR="/project/s1053/fv3core_serialized_test_data/${DATA_VERSION}/${experiment}"
-ARTIFACT_DIR="/project/s1053/performance/fv3core_monitor/${backend}"
+ARTIFACT_ROOT="/project/s1053/performance/"
+TIMING_DIR="${ARTIFACT_ROOT}/fv3core_monitor/${backend}"
+PROFILE_DIR="${ARTIFACT_ROOT}/fv3core_profile"
 CACHE_DIR="/scratch/snx3000/olifu/jenkins/scratch/store_gt_caches/${experiment}/${backend}"
+BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+
+# check if we store the results of this run
+if [[ "$BRANCH" != "master" ]]; then
```

Comment:
I would move that closer to the the definition of `SAVE_ARTIFACTS` in order to keep the logic in blocks

Reply:
done